include/jemalloc/internal/jemalloc_internal_overrides.h

```diff
@@ -0,0 +1,21 @@
+#ifndef JEMALLOC_INTERNAL_OVERRIDES_H
+#define JEMALLOC_INTERNAL_OVERRIDES_H
+
+/*
+ * Under normal circumstances this header serves no purpose, as these settings
+ * can be customized via the corresponding autoconf options at configure-time.
+ * Overriding in this fashion is useful when the header files generated by
+ * autoconf are used as input for another build system.
+ */
+
+#ifdef JEMALLOC_OVERRIDE_LG_PAGE
+    #undef LG_PAGE
+    #define LG_PAGE JEMALLOC_OVERRIDE_LG_PAGE
+#endif
+
+#ifdef JEMALLOC_OVERRIDE_LG_QUANTUM
+    #undef LG_QUANTUM
+    #define LG_QUANTUM JEMALLOC_OVERRIDE_LG_QUANTUM
+#endif
```

Comment:
Actually, the common case of LG_QUANTUM in defined in https://github.com/jemalloc/jemalloc/blob/3e2ba7a6510be583edb316372f8cfff35f2f25d5/include/jemalloc/internal/quantum.h (i.e. in internal_defs.h it's usually default to `/* #undef LG_QUANTUM */`) so this hack probably won't work for it. Maybe remove this for now?

Reply:
Sure, removed. I'll dig into this if I need to experiment with it in the future.